<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>379</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>367</width>
    <height>333</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Movie</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>351</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="MovieTitleLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
       </font>
      </property>
      <property name="text">
       <string>Movie Title:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="MovieTitleEntry"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="DateWatchedLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
       </font>
      </property>
      <property name="text">
       <string>Date Watched:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDateEdit" name="DateWatchedEntry">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2019</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2010</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumDate">
         <date>
          <year>2010</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="timeSpec">
         <enum>Qt::LocalTime</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="MovieRatingLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
       </font>
      </property>
      <property name="text">
       <string>Rating (1-10):</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="MovieRatingEntry">
        <property name="maxCount">
         <number>11</number>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="MovieGenreLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
       </font>
      </property>
      <property name="text">
       <string>Genre:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QComboBox" name="MovieGenreEntry">
        <item>
         <property name="text">
          <string>Drama</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Romance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Documentary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Animated</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fantasy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horror</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comedy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thriller</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Crime</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Western</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Adventure</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>War</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Biography</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sci-Fi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Musical</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="MovieLocationLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
       </font>
      </property>
      <property name="text">
       <string>Location Watched:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="MovieLocationHomeEntry">
        <property name="text">
         <string>Home</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="MovieLocationTheaterEntry">
        <property name="text">
         <string>Theater</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="MovieRewatchLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
       </font>
      </property>
      <property name="text">
       <string>Rewatch?</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QRadioButton" name="MovieRewatchEntryNo">
        <property name="text">
         <string>No</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="MovieRewatchEntryYes">
        <property name="text">
         <string>Yes</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="MovieCommentsLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
       </font>
      </property>
      <property name="text">
       <string>Comments:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPlainTextEdit" name="MovieCommentsEntry"/>
    </item>
    <item row="7" column="0">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="CancelEntryButton">
        <property name="text">
         <string>Cancel Entry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SubmitButton">
        <property name="text">
         <string>Submit Entry</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="EditMovieTitleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rubik Black</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Edit Movie</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
